brem/ui/RemoteServerSettingDialog.py

```diff
@@ -420,8 +420,8 @@ def setInfo(self, text):
         self.infolabel.setText(text)
 
     def authorize_key_worker(self, host, username, port, 
-                             progress_callback, message_callback):
-        a = brem.RemoteExecutionManager( logfile='generatekey.log', 
+                             progress_callback, message_callback, status_callback=None):
```

Comment:
If I'm not mistaken, the callbacks in the [`Worker`](https://github.com/paskino/qt-elements/blob/535e487d09d928713d7d6aa1123657597627c4b0/eqt/threading/QtThreading.py#L41-L43) are passed as the keyword parameters `progress_callback`, `message_callback` and `status_callback`.

So I suggest here to add `**kwargs` in the signature.